vms/volunteer/models.py

```diff
@@ -87,7 +87,7 @@ class Volunteer(models.Model):
         blank=True,
         validators=[
             RegexValidator(
-                r'^[(A-Z)|(a-z)|(\s)|(\.)|(\-)|(?)|(=)|(#)|(:)|(/)|(_)|(&)]+$',
+               r'^[0-9a-zA-Z\s\.\-?=#:/_&]+$',

```

Comment:
This will let me save ```a.a```, also a character followed by dot and then a number. Also many other invalid patterns. :(

Can you test with [this](http://www.rubular.com/r/utT4l6GQJa)?